modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -0,0 +1,233 @@
+package keeper_test
```

Comment:
This test file is copied from controller keeper. Didn't change any logic except calling ICAKeeper directly with icatypes.ControllerModuleName